packages/keychain/src/components/ErrorAlert.tsx

```diff
@@ -237,6 +237,7 @@ export function ControllerErrorAlert({
     case ErrorCode.AccountClassHashCalculation:
     case ErrorCode.AccountClassCompression:
     case ErrorCode.AccountFeeOutOfRange:
+    case ErrorCode.InsufficientBalance:
       title = "Account Error";
       break;
```

Comment:
I've updated this warning on later PR today. Pardon my late review 🙏 

https://github.com/cartridge-gg/controller/blob/c4d0e171f87fc768c5666d98423b91c6f693372b/packages/keychain/src/components/ErrorAlert.tsx#L182-L188